src/e2e/app.ts

```diff
@@ -511,6 +513,17 @@ export class App {
     return button.click();
   }
 
+  async selectDocumentationIdea(name: string): Promise<void> {
+    const document = await this.document();
+    const heading = await document.findByText(name, {
+      selector: "h3",
+    });
+    const handle = heading.asElement();
+    await handle!.evaluate((element) => {
+      element.parentElement?.click();
+    });
```

Comment:
It feels like there should be a better way but I failed to make one work in a reasonable amount of time.